test/ae_mdw_web/controllers/activities_controller_test.exs

```diff
@@ -767,29 +767,35 @@ defmodule AeMdwWeb.ActivitiesControllerTest do
                  "height" => ^height2,
                  "type" => "NameClaimEvent",
                  "payload" => %{
-                   "account_id" => ^account,
-                   "fee" => 333_333,
-                   "name" => ^plain_name,
-                   "name_fee" => 33333,
-                   "name_salt" => 3333,
-                   "nonce" => 333,
-                   "ttl" => 3_333_333,
-                   "type" => "NameClaimTx"
+                   "tx_hash" => "th_2CdVYuqtpcjoshDw2otjbLEyj8SpxjMP9MCgpn1oU9zGaqvUn4",
```

Comment:
can we pattern match please to avoid looking up for the hash?

Reply:
The `micro_time` could be easily tested the same way please.

Reply:
Done